lib/restaurantInfo.js

```diff
@@ -0,0 +1,17 @@
+const { collection } = require("./database");
+
+async function getRandomRestaurant(cuisine_value) {
+  const randomRestaurant = await collection("Restaurants")
+    .find({
+      cuisine: cuisine_value,
```

Comment:
Same here

Reply:
same here ...i'll put it on my to-do-list 💂🏻‍♀️